Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+ruby "2.4.1"
```

Comment:
@Rojo No for this particular project that I'm aware of. It broke my other programs. Just being conservative. OK, I'll do as you said.